src/wrappers/labels.ts

```diff
@@ -64,8 +65,20 @@ export default class {
       properties: ILabelProperties
     }[]
   ): Promise<ILabel[]> {
-    const pendingLabels = labels.map(r => this.create(r))
-    return await Promise.all(pendingLabels)
+    const pendingLabels = labels.map(r => {
+      return {
+        action: async () => {
+          return await this.create(r)
+        },
+        rollback: async (r?: ILabel) => {
+          if (r && r.id) {
+            this.delete(r.id)
+          }
+        },
+      }
+    })
+
+    return await saga(pendingLabels)
```

Comment:
shouldn't there be a type generic passed in here?

Reply:
double checked that it was properly inferred :)